web/src/shared/components/observed-element/observed-element.tsx

```diff
@@ -17,11 +17,11 @@ const useIntersectionThreshold = (
 };
 
 interface ObservedElementProps {
-    rootRef: React.RefObject<HTMLDivElement>;
+    rootRef?: React.RefObject<HTMLDivElement>;
     onIntersect: () => void;
     disabled?: boolean;
     width?: number;
-    height?: number;
+    height?: number | string;
```

Comment:
Mb here it is worth choosing one data type and, where it is set, coercioned to the required type?

Reply:
done